### PR TITLE
Spelling: UID

### PR DESCRIPTION
**Pull request by comradekingu**
Not sure about the second one.
There is also this instance of "Mail": "Allows login against an email attribute. Mail and mailPrimaryAddress will be allowed" which i didn't change.

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]

**Review by blizzz — CHANGES_REQUESTED** [review]
These refer to attribute names, and [uid](https://tools.ietf.org/html/rfc4519#section-2.39) is spelled lowercase (same for mail). In the first part you could put it into quotes instead, if you think it is less confusing.

**Comment by comradekingu** [comment]
@blizzz Like so?

File: apps/user_ldap/templates/part.wizard-loginfilter.php [hunk 1]
```diff
@@ -41,7 +41,7 @@
 			<textarea type="text" id="ldap_login_filter" name="ldap_login_filter"
 				class="ldapFilterInputElement"
 				placeholder="<?php p($l->t('Edit LDAP Query'));?>"
-				title="<?php p($l->t('Defines the filter to apply, when login is attempted. %%uid replaces the username in the login action. Example: "uid=%%uid"'));?>">
+				title="<?php p($l->t('Defines the filter to apply, when login is attempted. %%uid replaces the username in the login action. Example: "UID=%%uid"'));?>">
```
Comment (MorrisJobke) [inline apps/user_ldap/templates/part.wizard-loginfilter.php]: This also needs an update.

File: apps/user_ldap/templates/part.wizard-loginfilter.php [hunk 2]
```diff
@@ -9,7 +9,7 @@
 			</label>
 
 			<input type="checkbox" id="ldap_loginfilter_username"
-				   title="<?php p($l->t('Allows login against the LDAP / AD username, which is either uid or sAMAccountName and will be detected.'));?>"
+				   title="<?php p($l->t('Allows login against the LDAP / AD username, which is either "uid" or sAMAccountName and will be detected.'));?>"
```
Comment (blizzz) [inline apps/user_ldap/templates/part.wizard-loginfilter.php]: Then, sAMAccountName should also be in quotes.

File: apps/user_ldap/templates/part.wizard-loginfilter.php [hunk 3]
```diff
@@ -18,7 +18,7 @@
 			</label>
 
 			<input type="checkbox" id="ldap_loginfilter_email"
-				   title="<?php p($l->t('Allows login against an email attribute. Mail and mailPrimaryAddress will be allowed.'));?>"
+				   title="<?php p($l->t('Allows login against an email attribute. "mail" or mailPrimaryAddress is allowed.'));?>"
```
Comment (blizzz) [inline apps/user_ldap/templates/part.wizard-loginfilter.php]: Then, mailPrimaryAddress should also be in quotes. The "allowance" means both of them will be used.

**Review by blizzz — DISMISSED** [review]

File: apps/user_ldap/templates/part.wizard-loginfilter.php (hunk 3 above)
Comment (comradekingu) [inline apps/user_ldap/templates/part.wizard-loginfilter.php]: How about now?

File: apps/user_ldap/templates/part.wizard-loginfilter.php [hunk 4]
```diff
@@ -41,7 +41,7 @@
 			<textarea type="text" id="ldap_login_filter" name="ldap_login_filter"
 				class="ldapFilterInputElement"
 				placeholder="<?php p($l->t('Edit LDAP Query'));?>"
-				title="<?php p($l->t('Defines the filter to apply, when login is attempted. %%uid replaces the username in the login action. Example: "uid=%%uid"'));?>">
+				title="<?php p($l->t('Defines the filter to apply, when login is attempted. "%%uid" replaces the username in the login action. Example: "UID=%%uid"'));?>">
```
Comment (MorrisJobke) [inline apps/user_ldap/templates/part.wizard-loginfilter.php]: This one is still wrong 😢

**Review by MorrisJobke — CHANGES_REQUESTED** [review]

**Review by MorrisJobke — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**